go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/HdrHistogram/hdrhistogram-go v1.1.2
-	github.com/Shopify/sarama v1.33.0
+	github.com/Shopify/sarama v1.37.2
```

Comment:
How is it related to the zipkin exporter ?

Reply:
that's right. I spotted that. When I added otel zipkin exporter pkg, I ran `go mod tidy` which caused abv pkg upgraded.